pkg/sloop/store/untyped/db_utilities.go

```diff
@@ -0,0 +1,56 @@
+package untyped
+
+import (
+	"github.com/dgraph-io/badger/v2"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+)
+
+var collectSize = 100000
+
+func DropPrefixNoLock(keyPrefix []byte, db badgerwrap.DB) (error, float64) {
```

Comment:
you mentioned metrics - is it worthwhile to publish metrics around what this method sees?
number of keys to delete
number of keys deleted
total time to delete
average time to delete

Reply:
This is called from DoCleanup (garbage collection code) which emits both time and number of keys deleted metrics. I can add the metric for number of keys to delete.